api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -208,9 +205,8 @@ public void saveOrder_shouldNotSaveOrderIfOrderDoesntValidate() {
 		Order order = new Order();
 		order.setPatient(null);
 		order.setOrderer(null);
-		expectedException.expect(APIException.class);
-		expectedException.expectMessage("failed to validate with reason:");
-		orderService.saveOrder(order, null);
+		APIException exception = assertThrows(APIException.class, () -> orderService.saveOrder(order, null));
+		assertThat(exception.getMessage(), is("'Order. orderId: null patient: null concept: null care setting: null' failed to validate with reason: concept: No Concepts found - Invalid Concept Id, patient: Cannot be empty or null, encounter: Cannot be empty or null, orderer: Cannot be empty or null"));
```

Comment:
use matcher `containsString` with `"failed to validate with reason:"` like you did in other tests